backend/backend/game.py

```diff
@@ -188,3 +189,25 @@ def get_games(with_game_status=None):
         return result
     finally:
         conn.close()
+
+
+def get_this_game(player_id):
+    """Return the game that the player is currently playing in.
+
+    Arguments:
+        player_id: An int representing the player's id.
+
+    Returns:
+        A int representing the uid of the game the player is currently in.
+
+    """
+    all_games = get_games()
+    this_game_id = None
+
+    with Player(player_id) as player:
+        for game in all_games:
+            if player.username in all_games[game]:
+                this_game_id = game
+                break
+
+    return this_game_id
```

Comment:
This would be more efficient if we used a dedicated SQL query for it, I think – iterating over the games to find the right one seems wasteful. Do you think that’s something to change now or later? I’m happy with either, really.

Reply:
Flip, yeah of course, I didn't think of that. The efficiency probably won't be noticeable with the amount of games we'll have but it will look neater. I should be able to get that done soon.

Reply:
Well, the amount of games is the reason I’m easy – if it’s easy to do we might as well, but if not it’s not important yet.